docs/2_2_common_schema.adoc

```diff
@@ -864,24 +899,26 @@ calculated
 
 _[Note: (1) If `causality = "independent"`, it is neither allowed to define a value for `initial` nor a value for start.
 (2) If `causality = "input"`, it is not allowed to define a value for `initial` and a value for start must be defined.
-(3) If \(C) and `initial = "exact"`, then the variable is explicitly defined by its start value in Initialization Mode (so directly after calling `fmi3ExitInitializationMode`, the value of the variable is either the start value stored in element `<ScalarVariable><XXX start=YYY/>` or the value provided by `fmi3SetXXX`, if this function was called on this variable).]_
+(3) If \(C) and `initial = "exact"`, then the variable is explicitly defined by its start value in Initialization Mode (so directly after calling `fmi3ExitInitializationMode`, the value of the variable is either the start value stored in element `<Variable><XXX start=YYY/>` or the value provided by `fmi3SetXXX`, if this function was called on this variable).]_
```

Comment:
This should be `<Variable start="x"/>` (with double quotes because it cites an imaginary XML tag)

Reply:
I missed the newly introduced inheritance for variables here.
Not sure what you mean with double quotes here. Can you change it like you think it should be done?
Would this be ok: `<{Variable} start="x" />`
(see discussion above about imaginary xml tag)

Reply:
Actually this should be `{VariableType}` as a placeholder for Float64 etc.
To me the curly braces look a bit confusion but that's a matter taste.
We should define the placeholders in a central place (including all possible values).